distribution/ddf-common/src/main/resources/etc/system.properties

```diff
@@ -185,3 +185,8 @@ security.audit.roles=group,admin,manager,viewer,system-admin,system-history,syst
 # Disable Hazelcast version check
 #
 hazelcast.version.check.enabled=false
+
+#
+# Artemis Message Bus properties
+#
+artemis.amqp.port=5672
```

Comment:
Does SSL use the same port as the normal amqp port? How does the connection distinguish between doing SSL or not? Are we always forcing SSL?